test/system/301-rm.bats

```diff
@@ -3,13 +3,13 @@
 load helpers
 
 @test "Try to remove a nonexistent container" {
-  run_toolbox 1 rm nonexistentcontainer
-  is "$output" "toolbox: failed to inspect $todelete" "Toolbox should fail with: no such container"
+  run_toolbox rm nonexistentcontainer
```

Comment:
I'd probably merge the *Don't expect non-zero code on container removal* commit with the *update tests* commit, because that's what this is about - but still mention the reason in the commit message. :)